lldb/test/API/commands/statistics/basic/TestStats.py

```diff
@@ -209,6 +209,26 @@ def test_default_no_run(self):
             )
             self.assertGreater(module_stats["symbolsLoaded"], 0)
 
+    def test_default_no_run_no_preload_symbols(self):
+        """Test "statistics dump" without running the target and without
+        preloading symbols.
+
+        Checks that symbol count are zero.
+        """
+        # Make sure symbols will not be preloaded.
+        self.runCmd("settings set target.preload-symbols false")
+
+        # Build and load the target
+        self.build()
+        self.createTestTarget()
+
+        # Get statistics
+        debug_stats = self.get_stats()
+
+        # No symbols should be loaded in the main module.
+        main_module_stats = debug_stats["modules"][0]
+        self.assertEqual(main_module_stats["symbolsLoaded"], 0)
```

Comment:
rename to "symbolTableSymbolCount" per request in previous PR to keep naming consistent.